js/build.js

```diff
@@ -569,6 +569,16 @@ Fliplet.Widget.instance('chat', function (data) {
     var groupLabel = isChannel ? 'channel' : 'group';
     var isChannelOrGroup = isGroup || isChannel;
 
+    messages = _.filter(messages, function(message) {
+      if (message.data.conversationId === conversationId) {
+        _.remove(messagesIds, function(id) {
```

Comment:
FYI You can use this shorthand in the future.

```suggestion
        _.remove(messagesIds, { id: message.id });
```

Reply:
@tonytlwu, I know that this syntax is possible, but in this case the array has this format and the function is needed 
![изображение](https://user-images.githubusercontent.com/71251809/108378797-ff66ca80-720d-11eb-9403-109c68b6614c.png)